packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -205,23 +230,25 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
                 <TypingIndicator {...typingIndicatorProps} styles={typingIndicatorStyles} />
               )}
             </div>
-            <SendBox
-              {...sendBoxProps}
-              autoFocus={options?.autoFocus}
-              styles={sendBoxStyles}
-              /* @conditional-compile-remove(file-sharing) */
-              activeFileUploads={useSelector(fileUploadsSelector).files}
-              /* @conditional-compile-remove(file-sharing) */
-              onCancelFileUpload={adapter.cancelFileUpload}
-            />
-
-            {fileSharing?.uploadHandler && (
-              <FileUploadButton
-                accept={fileSharing?.accept}
-                multiple={fileSharing?.multiple}
-                onChange={fileUploadButtonOnChange}
-              />
-            )}
+            <Stack horizontal={formFactor === 'mobile'} horizontalAlign="start">
+              {formFactor === 'mobile' && (
+                <Stack verticalAlign="center">
+                  <AttachFileButton />
+                </Stack>
+              )}
+              <div style={{ width: '100%' }}>
```

Comment:
can this be a `StackItem` with `grow` property?
`width: 100%` is implying it should fill `100%` of the parents width but it really wants to be 100% minus any other sibling elements